app/services/idv/steps/in_person/state_id_step.rb

```diff
@@ -25,13 +25,13 @@ def call
           pii_from_user[:dob] = formatted_dob if formatted_dob
 
           if capture_secondary_id_enabled?
-            if pii_from_user[:same_address_as_id] == 'true'
+            if pii_from_user[:same_address_as_id].to_s == 'true'
               copy_state_id_address_to_residential_address(pii_from_user)
               mark_step_complete(:address)
             end
 
-            if initial_state_of_same_address_as_id == 'true' &&
-               pii_from_user[:same_address_as_id] == 'false'
+            if initial_state_of_same_address_as_id.to_s == 'true' &&
```

Comment:
Can we set initial_state_of_same_address_as_id  to boolean so we can eval it as a boolean here?

Reply:
You addressed this above. I will resolve.